src/App.js

```diff
@@ -1,12 +1,51 @@
+import React, { useState } from 'react';
+import QuoteApi from './components/NinjaAPI';
+import Contact from './components/Contact';
 import Calculator from './components/Calculator';
+import Footer from './components/Footer';
 import './Styles/Css/App.css';
 
 function App() {
-  return (
-    <div className="App">
-      <Calculator />
+  const [resourceType, setResourceType] = useState('Calculator');
+
+  const renderComponent = () => {
+    if (resourceType === 'QuoteApi') {
+      return <QuoteApi />;
+    }
+    if (resourceType === 'Contact') {
+      return <Contact />;
+    }
+    return <Calculator />;
+  };
```

Comment:
- [ ] Please ensure that the quotes are shown on the same page as the calculator. This would mean importing the `<QuoteApi />` component without the `if` condition that you have placed based on the value of `resourceType`.

Reply:
uhu I see, I thought I had to display it in a different page, will change it right away. thanks